core/storage/trie/serialization/trie_serializer_impl.cpp

```diff
@@ -68,6 +71,11 @@ namespace kagome::storage::trie {
 
   outcome::result<PolkadotTrie::NodePtr> TrieSerializerImpl::retrieveNode(
       const std::shared_ptr<OpaqueTrieNode> &parent) const {
+    if (auto p = std::dynamic_pointer_cast<DummyValue>(parent); p != nullptr) {
```

Comment:
Is this if identical to the if below?

Reply:
- https://github.com/soramitsu/kagome/issues/1470